torch/utils/data/datapipes/iter/selecting.py

```diff
@@ -24,7 +23,6 @@ class FilterIterDataPipe(IterDataPipe[T_co]):
     Args:
         datapipe: Iterable DataPipe being filtered
         filter_fn: Customized function mapping an element to a boolean.
-        drop_empty_batches (Deprecated): By default, drops a batch if it is empty after filtering instead of keeping an empty list
```

Comment:
I have a question regarding `drop_empty_batches`. Should we change the behavior as `drop_empty_batches` as False? Users should be able to filter out the empty batches/data easily with a `filter_fn`. And, what if some users want to keep empty batches in the pipeline, they won't have a chance to do so.

WDYT?
cc: @VitalyFedyunin @NivekT

Reply:
Yea, I think if we get rid of this option, the default should be do nothing rather than forcibly dropping empty batches.